src/v0/destinations/customerio/util.js

```diff
@@ -236,13 +237,14 @@ const defaultResponseBuilder = (message, evName, userId, evType, destination, me
   // DEVICE registration
   const isDeviceRelatedEvent = isdeviceRelatedEventName(evName, destination);
   if (isDeviceRelatedEvent && userId && token) {
+    const timestamp = message.originalTimestamp;
```

Comment:
why a direct reference of originalTimestamp ? We use mapping config first to check on timestamp & then only fallback to originalTimestamp in all other destinations

Reply:
Incorporated